### ISSUE TITLE
SearXNG does not returns some results from bing and duckduckgo

### ISSUE THREAD

**Issue by tubbadu** [searxng/searxng#2666]
**Version of SearXNG, commit number if you are using on master branch and stipulate if you forked SearXNG**
2023.8.21+d52919cc (https://search.ononoki.org/search)

**What happened?**
some results that search engines display are not displayed by searxng. For instance, searching `scriptinator` on bing or duckduckgo returns many results, but with searxng (with bing and duckduckgo enabled) returns 0 results. For other queries it works instead.

**How To Reproduce**
Go to https://search.ononoki.org/, enable both bing and duckduckgo and search for `scriptinator` (other "strange" queries have the same behavior)

**Expected behavior**
the same results obtained from bing.com and duckduckgo.com

**other informations**
I'm doing this bing and duckduckgo, but probably other engines behave the same. Interesting fact, Startpage works (google is blocked by a "too many requests" error so I can't tell for it)

**Comment by Bnyro** [comment]
Partially a duplicate of #2641 I think?

**Comment by return42** [comment]
> Partially a duplicate of https://github.com/searxng/searxng/issues/2641 I think?

Yes, we temporary blocked by ddg's and this engines bot detection:

![image](https://github.com/searxng/searxng/assets/554536/d1aa40d0-26f3-4b7a-94bf-cfdf562e1621)

disband these blockades needs deep reverse engineering .. with open end / its not promised there is a solution for each of theses engines .. sadly I hadn't had time for more  reverse engineering :(

**Comment by return42** [comment]
FWI I hope ddg works now better since 

- https://github.com/searxng/searxng/pull/2716/

fixes the vqd value / this value is used by engines `duckduckgo` and `duckduckgo_images`.

**Comment by return42** [comment]
Please let us open (or contribute to) issues dedicated to an engine / we can not track issues over a mix of engines without ending in confusion and aberrations.

- https://github.com/searxng/searxng/issues/2616
- https://github.com/searxng/searxng/issues/2719
- https://github.com/searxng/searxng/issues/2641
- https://github.com/searxng/searxng/issues/2456#issuecomment-1618000053
- https://github.com/searxng/searxng/issues/1081#issuecomment-1488162918
- [Answer CAPTCHA from server’s IP](https://docs.searxng.org/admin/answer-captcha.html#answer-captcha-from-server-s-ip)
